src/app/static/src/app/components/baseline/Baseline.vue

```diff
@@ -39,22 +47,39 @@
     import {BaselineState} from "../../store/baseline/baseline";
     import FileUpload from "../FileUpload.vue";
 
+    interface PartialFileUploadProps {
+        valid: Boolean,
+        error: string,
+        existingFileName: string,
+    }
+
     const namespace: string = 'baseline';
 
     export default Vue.extend({
         name: "Baseline",
         computed: mapState<BaselineState>(namespace, {
             country: state => state.country,
-            pjnzError: state => state.pjnzError,
-            pjnzFileName: state => state.pjnzFilename,
-            validShape: state => state.shape != null,
-            shapeFileName: state => state.shape && state.shape.filename,
-            shapeError: state => state.shapeError
+            pjnz: state => ({
```

Comment:
Yeah, this is nice, more readable than before. Worth pulling out the interface and putting it into surveyAndProgramme to, or do that in a separate ticket?